src/Period/MediumPeriodHTMLFormatter.php

```diff
@@ -12,12 +12,18 @@
 
 class MediumPeriodHTMLFormatter implements PeriodFormatterInterface
 {
+    private $locale;
+
+    public function __construct($locale)
```

Comment:
Old default value is missing here, this would be a BC break causing issues for existing uses of the library.